config/webpack/webpack.config.js

```diff
@@ -88,29 +95,27 @@ const devConfig = (options) => {
       },
     },
     output: {
-      path: path.join(rootPath, 'build'),
+      filename: `${filename}.js`,
+      path: outputPath,
+      publicPath,
     },
-    devtool: 'cheap-source-map',
+    devtool: production ? undefined : 'cheap-source-map',
```

Comment:
Instead of having the ternary set it to undefined here, can we set it inside the if statement you have below? I feel that not having a key when we don't have a value seems more robust.